docs/cloud-edge/modules/user-agent-filter.md

```diff
@@ -0,0 +1,135 @@
+# User Agent Filter
+
+---
+
+## Overview
+
+User Agent Filter module allows you to allow or deny traffic based on the user-agent request header. You can define allow and deny regular expression rules that apply to the incoming user-agent request header.
```

Comment:
User Agent Filter module --> The User Agent Filter module

I suggest keeping the markup consistent throughout the document:
define allow and deny --> define `allow` and `deny`